src/NzbDrone.Core/Notifications/Join/JoinSettings.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections.Generic;
+using FluentValidation;
+using NzbDrone.Core.Annotations;
+using NzbDrone.Core.ThingiProvider;
+using NzbDrone.Core.Validation;
+
+namespace NzbDrone.Core.Notifications.Join
+{
+    public class JoinSettingsValidator : AbstractValidator<JoinSettings>
+    {
+        public JoinSettingsValidator()
+        {
+            RuleFor(c => c.APIKey).NotEmpty();
+        }
+    }
+
+    public class JoinSettings : IProviderConfig
+    {
+        private static readonly JoinSettingsValidator Validator = new JoinSettingsValidator();
+
+        [FieldDefinition(0, Label = "API Key", HelpText = "The API Key from your Join account settings (click Join API button): https://joinjoaomgcd.appspot.com/", HelpLink = "https://joinjoaomgcd.appspot.com/")]
```

Comment:
We don't need to add the URL here as the HelpLink field will be present (and they won't try to copy the URL out of the tooltip awkwardly).